C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackAdapter.cs

```diff
@@ -248,15 +264,15 @@ public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext
         /// <param name="activity">The updated activity in the form `{id: <id of activity to update>, ...}`</param>
         public override async Task<ResourceResponse> UpdateActivityAsync(ITurnContext turnContext, Activity activity, CancellationToken cancellationToken)
         {
-            ResourceResponse results = null;
+            dynamic results = null;
```

Comment:
Why is `results` dynamic? Isn't it a response from `UpdateAsync`? Which I assume that is `ResourceResponse`.
And if that is correct, cant we create `results` as `ResourceResponse`?

Same with `message`.